tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -205,4 +205,24 @@ class TargetedSuite extends SemanticdbSuite {
 
     }
   )
+
+  targeted(
```

Comment:
I kept this test to ensure that it runs in 2.11/2.12/2.13. The  expect tests only run in 2.12